packages/techdocs-cli/src/lib/httpServer.ts

```diff
@@ -59,6 +59,16 @@ export default class HTTPServer {
       const proxyHandler = this.createProxy();
       const server = http.createServer(
         (request: http.IncomingMessage, response: http.ServerResponse) => {
+          // This endpoind is used by the frontend to issue a cookie for the user.
+          // It is necessary because the MkDocs server doesn't expose it.
+          if (request.url === '/api/techdocs/.backstage/v1-cookie') {
```

Comment:
```suggestion
          if (request.url === '/api/techdocs/.backstage/auth/v1/issue-cookie') {
```

was it something more like this that we talked about? i'm not sure anymore :)

Reply:
also is it intentional that it doesn't do the actual issuance? i guess it is, just checking

Reply:
I think it should be `/cookie` because it is the URL we request in the hook here:
https://github.com/backstage/backstage/blob/master/plugins/auth-react/src/hooks/useCookieAuthRefresh/useCookieAuthRefresh.tsx#L40

But for some reason when I run it locally, it tries to request /.backstage/v1-cookie
![image](https://github.com/backstage/backstage/assets/6290749/640f0a35-4456-46cd-8ecd-c683f85c973d)

I'll try to cleanup my local branch just to make sure

Reply:
Running only the cli app, it is calling the `/cookie` path, so I updated the path to "/cookie"